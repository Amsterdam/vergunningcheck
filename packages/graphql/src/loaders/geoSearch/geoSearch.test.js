require("@spotify/polly-jest-presets");
const { loaderFunctionsTest } = require("../../testUtils");
const geoSearch = require(".");

const key1 = "121613 487990";
const key2 = "121263 484745";

describe("GeoSearch loader", () => {
  describe("Basic functions", loaderFunctionsTest(geoSearch, key1, key2));
});
