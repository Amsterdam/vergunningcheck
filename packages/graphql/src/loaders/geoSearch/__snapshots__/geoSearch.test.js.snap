// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GeoSearch loader Basic functions module result 1`] = `
Array [
  Array [
    Object {
      "_type": "CityScape",
      "id": "1477",
      "name": "Amsterdam - Binnen de Singelgracht",
    },
  ],
]
`;

exports[`GeoSearch loader Basic functions module result 2`] = `
Array [
  Array [
    Object {
      "_type": "CityScape",
      "id": "1672",
      "name": "Amsterdam - Plan Zuid",
    },
  ],
]
`;
